### PR TITLE
#181339488 user login page

### PR DESCRIPTION
**Pull request by Gabin-ishimwe**
#### What does this PR do?
user login page
#### Description of Task to be completed?

- user can login to barefoot application
- validate user inputs 

#### How should this be manually tested?
here is the deployed link: [https://deploy-preview-10--barefoot-nomad-fe.netlify.app/login](https://deploy-preview-10--barefoot-nomad-fe.netlify.app/login)
#### Any background context you want to provide?

- you should input a valid email with `@` in it
- you should input a password of at least 6 characters with at least one capital letter and one number

#### What are the relevant pivotal tracker stories?
[181339488](https://www.pivotaltracker.com/n/projects/2555280/stories/181339488)
#### Screenshots (if appropriate)
image
image

#### Questions:

### CODE REVIEW

**Comment by ncutixavier** [comment]
Your test coverage is good for now. Just work on Axios interceptor, you may refer to this link https://axios-http.com/docs/interceptors

File: src/redux/actions/auth.js [hunk 1]
```diff
@@ -1,10 +1,50 @@
...
+  return {
+    type: ERROR_LOGIN,
+    payload: error,
+    isLogged: false,
+    loading: false,
+  };
+};
+
+export const loginAction = ({ email, password }) => {
+  return async (dispatch) => {
```
Comment (Code-250) [inline src/redux/actions/auth.js:26]: if you are going to use async consider await using await not .then with async

**Review by Code-250 — COMMENTED** [review]
do not mix async await with .then and .catch promises since both do the same thing consider using one of them.

**Review by Eli250 — APPROVED** [review]

**Review by Code-250 — APPROVED** [review]

**Review by ncutixavier — CHANGES_REQUESTED** [review]

**Comment by ncutixavier** [comment]
image
Show sign-out only when user logged in

**Comment by ncutixavier** [comment]
Also work on your footer

**Comment by ncutixavier** [comment]
Have you forgotten the naming convention? because It seems like your PR title does not format well and your PR description doesn't look well

**Comment by Gabin-ishimwe** [comment]
> Have you forgotten the naming convention? because It seems like your PR title does not format well and your PR description doesn't look well
sorry i forgot about it but now it is changed and also changed the PR description

**Review by ncutixavier — APPROVED** [review]

**Pull request merged**